docs/releases/v2018.1.rst

```diff
@@ -0,0 +1,374 @@
+========================
+
+The *gluon-ebtables-limit-arp* (*ebtables-limit-arp* feature flag) package adds filters to limit the
+amount of ARP requests client devices are allowed to send into the mesh.
+
+Certain client applications are known to generate a significant amount of such ARP requests and
+are reportedly becoming more and more common. Without this package, such clients are one
+known cause for mesh wide load and congestion problems (see also the :ref:`releases-v2018.1-known-issues`
+section below). It is therefore recommended to install this package via the site.mk.
+
+Due to the general unfitness of ebtables for frequent dynamic updates and the therefore
+unconventional approach implemented by this package it is not added by default yet.
+Feedback, especially with a close look on load and congestion on nodes with a large
+number of changing client devices, is very much welcome. Depending on the feedback,
+we might enable this feature by default in a future release.
+
+Public key in respondd data (optional)
+======================================
+
+If desired, the fastd public key of a node can be included in the respondd nodeinfo data,
+faciliating the correlations of VPN peers and nodes. As the VPN key is transmitted unencrypted
+in the fastd handshake, this would theoretically allow an ISP to determine which nodes
+are operated behind which internet line. Therefore, this feature must be enabled explicitly
+by setting *mesh_vpn.pubkey_privacy* to ``false`` in *site.conf*.
+
+B.A.T.M.A.N. V (experimental)
+=============================
+
+When using batman-adv compat 15, it is now possible to switch to the new routing
+algorithm B.A.T.M.A.N. V (while the old algorithm is called B.A.T.M.A.N. IV) by
+setting *mesh.batman_adv.routing_algo* to ``"BATMAN_V"``. Note that the new routing
+algorithm is not backwards-compatible, so nodes using different algorithms can
+not interoperate.
+
+.. _releases-v2018.1-site-changes:
+
+Site changes
+************
+
+site.mk
+=======
+
+* Due to improved package dependency handling, the packages
+  *gluon-config-mode-core* and *gluon-setup-mode* do not need
+  to be listed explicitly in *site.mk* anymore; they will be
+  pulled in implicitly.
+* As mentioned above, we recommend to include the *ebtables-limit-arp*
+  feature flag.
+* We recommend to use *GLUON_FEATURES* for all Gluon packages, and rely on
+  *GLUON_SITE_PACKAGES* for non-Gluon (OpenWrt) package only, as explained
+  in :ref:`user-site-feature-flags`.
+
+site.conf
+=========
+
+When updating a site configuration from Gluon 2017.1.x, the following changes
+must be made:
+
+* .. code-block:: lua
+
+    domain_seed = 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx',
+
+  These 32 bytes of random data (encoded in hexadecimal) are used to seed a number
+  of site/domain specific random values that must be the same on all nodes of the
+  same mesh, but different for different meshes. The following command
```

Comment:
maybe instead of `different for different meshes` write `unique for different meshes`